pom.xml

```diff
@@ -107,33 +115,33 @@
             <artifactId>scala-compiler</artifactId>
             <version>${scala.version}</version>
             <scope>test</scope>
-         </dependency>
+        </dependency>
 
         <dependency>
```

Comment:
Why do we need to change the versions of these dependencies? Is it related to the changes for Spark/Scala?

Reply:
Making it work with scala 2.12